docs/internals/contributing/writing-documentation.txt

```diff
@@ -606,15 +606,7 @@ example:
 
     Default: ``[]`` (Empty list)
 
-    A list of all the people who get code error notifications. When
-    ``DEBUG=False`` and a view raises an exception, Django will email these people
-    with the full exception information. Each member of the list should be a tuple
-    of (Full name, email address). Example::
-
-        [("John", "john@example.com"), ("Mary", "mary@example.com")]
-
-    Note that Django will email *all* of these people whenever an error happens.
-    See :doc:`/howto/error-reporting` for more information.
+    A list of all the people who get code error notifications...
```

Comment:
took me a second to realize the `...` is just intentionally eliding the full message (there are other code snippets in this document that are doing the same thing), so it seems intentional.

This is just an example of how to use the `.. setting::` directive, not the actual settings for `ADMINS`.

Reply:
Yeah, I was grepping for ADMINS and MANAGERS, and was surprised to find an (outdated) copy of the ADMINS setting in the "Writing Documentation" section. Since it was already out of sync with the actual settings docs, seemed better to just show an excerpt here.